app/views/employees/_form.html.erb

```diff
@@ -0,0 +1,33 @@
+<%= form_with(model: employee) do |form| %>
+  <% if employee.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(employee.errors.count, "error") %> prohibited this employee from being saved:</h2>
+
+      <ul>
+        <% employee.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div class="field">
+    <%= form.label :name %>
+    <%= form.text_field :name %>
+  </div>
+
+  <div class="field">
+    <%= form.label :email %>
+    <%= form.text_field :email %>
+  </div>
+
+  <div class="field">
+    <%= form.label :private_number %>
+    <%= form.number_field :private_number %>
+  </div>
+
+
+  <div class="actions">
+    <%= form.submit %>
+  </div>
```

Comment:
I recommend that you style the forms.

Reply:
The styles for all this scaffold elements are going to be in a different issued.